Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -216,7 +214,7 @@ void InterfacePane::LoadConfig()
       m_combobox_theme->findText(QString::fromStdString(SConfig::GetInstance().theme_name)));
 
   const QString userstyle = Settings::Instance().GetCurrentUserStyle();
-  const int index = m_combobox_userstyle->findText(QFileInfo(userstyle).baseName());
+  const int index = m_combobox_userstyle->findText(QFileInfo(userstyle).completeBaseName());
```

Comment:
I know little about this but is your thought that `baseName()` might be truncating the style a little too much, ex:

`mycoolstyle.123.css`

`baseName()` = `mycoolstyle`
`completeBaseName()` = `mycoolstyle.123`

Reply:
Yes, exactly. Though I guess this `baseName` -> `completeBaseName` change isn't entirely related to the main point of this PR.

Reply:
Actually, it seems like I forgot to push a change I made to this line... I'm going to do that right now.

Reply:
Thanks.  I was also questioning it for the other line that uses `completeBaseName` but that explanation makes sense I think.

Reply:
For what it's worth, for that other line, `completeBaseName` matches the behavior of the previously used `SplitPath` method.